cmd/talosctl/cmd/mgmt/validate.go

```diff
@@ -56,5 +67,6 @@ func init() {
 		fmt.Sprintf("the mode to validate the config for (valid values are %s, %s, and %s)", runtime.ModeMetal.String(), runtime.ModeCloud.String(), runtime.ModeContainer.String()),
 	)
 	cli.Should(validateCmd.MarkFlagRequired("mode"))
+	validateCmd.Flags().BoolVarP(&validateStrictArg, "strict", "", false, "treat validation warnings as errors")
 	addCommand(validateCmd)
 }
```

Comment:
Do we have tests or scripts where we run `talosctl validate` so this code can be tested?